go/cmd/vtctldclient/command/backups.go

```diff
@@ -35,15 +35,15 @@ import (
 var (
 	// Backup makes a Backup gRPC call to a vtctld.
 	Backup = &cobra.Command{
-		Use:                   "Backup [--concurrency <concurrency>] [--allow-primary] <tablet_alias>",
+		Use:                   "Backup [--concurrency <concurrency>] [--allow-primary] [--upgrade_safe] <tablet_alias>",
```

Comment:
All new flags should use dashes rather than underscores. It looks like the flag actually is using a dash, but the use line is wrong.